deploy/README.md

```diff
@@ -0,0 +1,31 @@
+# Basic Deployment
+
+## Tools
+
+1. [Capistrano](http://capistranorb.com/)
+2. [Ansible](https://www.ansible.com/)
+
+## Servers
+
+1. [Nginx](https://nginx.org)
+2. [Puma](http://puma.io/)
+
+## Managers
+
+1. [RVM](https://rvm.io/)
+2. [NVM](https://github.com/creationix/nvm)
```

Comment:
`rbenv` better fits into 'shell' when you switching between projects its always know what version of ruby to use... and its more light. With `rvm` a had many problems with env configs